src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsTransportFactory.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FakeSms;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+/**
+ * @author James Hemery <james@yieldstudio.fr>
+ */
+final class FakeSmsTransportFactory extends AbstractTransportFactory
+{
+    protected $serviceProvider;
+
+    public function __construct(ServiceProviderInterface $serviceProvider)
+    {
+        parent::__construct();
+
+        $this->serviceProvider = $serviceProvider;
+    }
+
+    /**
+     * @return FakeSmsMailTransport|null
```

Comment:
```suggestion
     * @return FakeSmsMailTransport
```

This will never return `null`